projects/1.18/assets/chileancraft/chileancraft/lang/zh_cn.json

```
@@ -0,0 +1,871 @@
+{
+  "advancements.alabordaje.descr": "获得一柄水手的佩剑",
+  "advancements.alabordaje.title": "上船吧，孩子！",
+  "advancements.estamos_bien_los_33.descr": "遇见一名迷路的矿工",
+  "advancements.estamos_bien_los_33.title": "他大概33岁",
```

```suggestion
  "advancements.estamos_bien_los_33.title": "我们33个人都很好",
```
https://www.infobae.com/2010/10/12/1008539-33-el-numero-del-milagro-chileno/